.github/workflows/lint-test.yaml

```diff
@@ -57,6 +57,12 @@ jobs:
       - name: Run chart-testing (install)
         run: ct install --target-branch main
 
+      - name: Run daemonset and deployment install test
+        run: |
+          kubectl apply -f ./charts/opentelemetry-collector/examples/daemonset-and-deployment/rendered
+          kubectl rollout status daemonset example-opentelemetry-collector-agent --timeout=30s
+          kubectl rollout status deployment example-opentelemetry-collector --timeout=30s
```

Comment:
I think we probably need to extract chart specific steps to separate CI jobs

Reply:
https://github.com/open-telemetry/opentelemetry-helm-charts/issues/179

Reply:
Want that to be part of this PR, or can this be merged and the. The breakout happen in a different PR

Reply:
Yes, that's what I assumed